.gitignore

```diff
@@ -7,6 +7,7 @@ vendor
 Gemfile.lock
 bin
 .rvmrc
+.tool-versions
```

Comment:
We don't mind adding this line, though.

I, however, prefer adding those to my [global git config](https://github.com/pirj/dotfiles/blob/c358379ce0b7a3675920d1309f8649f30814c36f/.config/git/ignore#L7).